cluster-autoscaler/FAQ.md

```diff
@@ -1062,6 +1062,17 @@ git pull origin machine-controller-manager-provider
 
 #### Step 2:
 
+Merge master onto the `machine-controller-manager-provider` branch:
+```
+git merge master
+```
+
+Resolve the rebase-conflicts by appropriately accepting the incoming changes or the current changes.
```

Comment:
Shouldn't this be merge conflict?

Reply:
Also, in Step 1, line 1057 where we grep on 'Updating vendor against' is not needed anymore.

Reply:
> Also, in Step 1, line 1057 where we grep on 'Updating vendor against' is not needed anymore.

This will be needed for the part `syncing with kubernetes/autoscaler < v1.21.0` because in that case old `update_vendor` script is used.
Since this bifurcation of `syncing with based on upstream autoscaler version` will always be there in FAQ(irrespective of g/autoscaler version) , so thats why I kept it.